src/core/init/media_source_content_initializer.ts

```diff
@@ -356,7 +356,7 @@ export default class MediaSourceContentInitializer extends ContentInitializer {
         if (initCanceller.isUsed()) {
           return;
         }
-        triggerEvent("reloadingMediaSource", null);
+        triggerEvent("reloadingMediaSource", autoPlay);
```

Comment:
Nice, I had more or less the same solution when I recently wrote #1253.

Though I added it as an object (and though this is not at all needed, alongside the `position` property), because I thought that it made more sense purely in the sense of readability to have a `reloadingMediaSource` event linked to various named properties which describes how the reload is configured than just to a boolean which is hard to make sense of directly.

What do you think?

Reply:
Definitely makes sense, I'll adapt to keep it uniform!